spec/spec_helper.rb

```diff
@@ -1,5 +1,4 @@
 require 'factory_bot'
-require 'database_cleaner'
```

Comment:
How do you clean up when testing locally if you remove this? 

I assume that after removing this and a test run you have data from the factories in your test database? 

Am i missing a place where you recreate / clear the database before each spec run?

Reply:
No, RSpec rolls back its own transactions.

Reply:
Does it by default? https://relishapp.com/rspec/rspec-rails/docs/transactions

Reply:
Looks like this is included in https://github.com/zooniverse/tove/blob/31a81bb3058d2c8923ce619096e530702fddf62e/spec/rails_helper.rb#L41 not spec_helper

Reply:
Yeah, installing rspec puts that line in `rails_helper.rb`, just like those docs say. Both are required by `.rspec`, so it's being set for all tests.